assets/js/components/Dashboard/PlaceRow.tsx

```diff
@@ -151,6 +151,25 @@ const PlaceRow = (props: PlaceRowProps): JSX.Element => {
     )?.inlineMap;
   };
 
+  const formatStationName = (stationName: string) => {
+    let isTerminalStop = false;
+    if (props.filteredLine && props.filteredLine !== null) {
```

Comment:
```suggestion
    if (props.filteredLine) {
```
^ this verifies filteredLine exists and is truthy

Reply:
If the only thing that impacts whether a stop is capitalized / not capitalized is that they are a terminal stop, and this value is hardcoded in `stationOrder.ts`, then I'm not sure why the if/else is needed

Reply:
I could have sworn that wasn't working at one point... Definitely is the right way to do it though.

As for the reason for the if/else, it was mostly to avoid terminal stop lookups in lines that are not currently in the filter. If we are filtering on RL, we don't need to know the terminal stops for GL. Does that make sense?